scripts/MantidIPython/plot_functions.py

```diff
@@ -0,0 +1,111 @@
+"""
+Plotting functions for use in IPython notebooks that are generated by MantidPlot
+"""
+
+import matplotlib.pyplot as plt
+import sys
+import os
+
+# Import Mantid
+CURRENT_DIR = os.getcwd()
```

Comment:
Ok. As long as there is an item to clean it up when the launchers get created then we can leave it in for now. Could you add a comment to that affect though with a link to the issue?